src/transformStateWithClones.js

```diff
@@ -65,7 +65,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  let newState = { ...state };
+  const cloneState = [];
```

Comment:
It`s better to use another name of the variable for massive, like "states" for example. It`s because your newState is actually the state clone